env/src/main/java/com/sap/cloud/security/config/ServiceBindingEnvironment.java

```diff
@@ -75,22 +73,26 @@ public ServiceBindingEnvironment withEnvironmentVariableReader(UnaryOperator<Str
 	 * <li>BROKER</li>
 	 * <li>SPACE</li>
 	 * <li>DEFAULT</li>
+	 * <li>&lt;other or missing plan&gt;</li>
```

Comment:
Good catch. I added it because there was some unit test failing that led me to believe it has to support more than the four known plans. I removed it now and it looks like tests are running fine.